container/cypress/e2e/test-app/iframe/iframe-container.cy.js

```diff
@@ -23,6 +23,32 @@ describe('Iframe Container Test', () => {
       });
   });
 
+  it('sendCustomMessage', () => {
+    cy.get(containerSelector)
+      .shadow()
+      .get('iframe')
+      .then(iframe => {
+        const $body = iframe.contents().find('body')
+        cy.wrap($body)
+          .find('#content')
+          .should('have.text', ' ');
+      });
+
+    cy.get('#btn-1')
+      .click()
+      .then(() => {
+        cy.get(containerSelector)
```

Comment:
nested **cy.get('#btn-1')** > **cy.get(containerSelector)** is not a good approach. 

You can do it step by step. Don't have to nest them like this.